protocol/rpcconsumer/rpcconsumer_server.go

```diff
@@ -811,3 +879,11 @@ func (rpccs *RPCConsumerServer) appendHeadersToRelayResult(ctx context.Context,
 	}
 	relayResult.Reply.Metadata = append(relayResult.Reply.Metadata, metadataReply...)
 }
+
+func (rpccs *RPCConsumerServer) IsHealthy() bool {
+	if RelaysHealthEnable {
+		return rpccs.relaysMonitor.IsHealthy()
```

Comment:
now you dont need the if. 
```go
func (rm *RelayMonitor) IsHealthy() bool {
   if rm == nil {
           return false
       }
       return .......
}

```

Reply:
0d17caa346eb14e794b6c77db0b92a0a533c75d1